src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -325,9 +325,15 @@ bool StridedSlice::needShapeInfer() const {
 }
 
 void StridedSlice::execute(dnnl::stream strm) {
-    if (!execPtr)
-        OPENVINO_THROW(errorPrefix, "doesn't have compiled executor!");
-
+    if (!execPtr) {
+        if (!isDynamicNode() && !hasConstAttrInputs) {
+        // SliceScatter due to not having data dependency on shape may not call prepareParams when start/stop/step values are non-constant in Static execution.
+        // In Slice and SliceScatter op, prepareParams would be called by createPrimitive (if const inputs) or by updateDynamicParams in case of dynamic node.
+        StridedSlice::prepareParams();
+        } else {
+            OPENVINO_THROW(errorPrefix, "doesn't have compiled executor!");
+        }
+    }
```

Comment:
If the data on the corresponding inputs are not constant, it means that the executor must be reinitialized every inference call, as the data in variables may be updated each inference. If the tests pass, that means that either the test converge is still not sufficient, or there is no real data dependency and the executor may be created at the compilation stage.